src/lib/shady-render.ts

```diff
@@ -33,17 +33,20 @@ declare global {
 const getTemplateCacheKey = (type: string, scopeName: string) =>
     `${type}--${scopeName}`;
 
-const verifyShadyCSSVersion = () => {
+let verifyShadyCSSVersion = () => {
```

Comment:
Let's just change this away from a function to a const boolean. I don't immediately see a reason we shouldn't do the check at boot up.